packages/babel-parser/test/fixtures/core/escape-string/invalid-decimal-escape-strict-directive-function/input.js

```diff
@@ -20,13 +20,3 @@ function d() {
   "\5";
 }
 
-function c() {
-  "use strict";
-  05;
-}
-
-function d() {
-  "use strict";
-  04;
-  05;
-}
```

Comment:
They are moved to https://github.com/babel/babel/pull/12366/files#diff-cd40ee2fb191cbf29a8bbfc49ae7471de322ff395f79acf129d3f9cd3832a688

So this test is focused on numeric escape sequence only.